website/src/common/Footer.jsx

```diff
@@ -0,0 +1,111 @@
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import React from "react";
+
+import {
+  AiFillGithub,
+  AiFillSlackCircle,
+  AiFillTwitterCircle,
+} from "react-icons/ai";
+import { BsStackOverflow } from "react-icons/bs";
+import { Link } from "react-router-dom";
+
+const Footer = () => {
+  const socials = [
+    { url: "https://github.com/vitessio/arewefastyet", icon: AiFillGithub },
+    { url: "https://vitess.io/slack", icon: AiFillSlackCircle },
+    { url: "https://twitter.com/vitessio", icon: AiFillTwitterCircle },
+    { url: "https://stackoverflow.com/search?q=vitess", icon: BsStackOverflow },
+  ];
+
+  const items = [
+    {
+      title: "Benchmarks",
+      links: [
+        { title: "Daily", to: "/daily" },
+        { title: "Compare", to: "/compare" },
+        { title: "Micro", to: "/micro" },
+        { title: "Macro", to: "/macro" },
+        { title: "PR", to: "/pr" },
+      ],
+    },
+    {
+      title: "Vitess",
+      links: [
+        { title: "Docs", to: "https://vitess.io/docs" },
+        { title: "Blog", to: "https://vitess.io/blog" },
+        { title: "Community", to: "https://vitess.io/community" },
+      ],
+    },
+    {
+      title: "arewefastyet",
+      links: [{ title: "github", to: "https://www.github.com/vitessio/arewefastyet" }],
+    },
+  ];
+
+  return (
+    <footer className="p-page relative mt-10 py-20 text-back">
+      <div className="absolute bottom-6 left-6 right-6 top-6 -z-1 rounded-lg bg-primary" />
+      <div className="flex justify-between text-sm font-light tracking-tight">
+        <div
+          className={`flex basis-[20%] flex-col items-center gap-y-5 opacity-90 drop-shadow-[0px_0px_0px_theme("colors.back")]`}
+        >
+          <img
+            src="/logo.png"
+            alt="agrosurance logo"
+            className="aspect-square w-1/3"
+          />
+          <h2 className="text-xl font-medium">arewefastyet</h2>
+        </div>
+        {items.map((item, key) => (
+          <div key={key} className="flex flex-col">
+            <h5 className="font-semibold">{item.title}</h5>
+            <div className="my-7 flex flex-col gap-y-3">
+              {item.links.map((link, key) => (
+                <Link key={key} to={link.to}>{link.title}</Link>
+              ))}
+            </div>
+          </div>
+        ))}
+
+        <div className="flex flex-col items-center text-center text-back text-opacity-80">
+          <h5 className="my-3 text-4xl font-bold tracking-tighter text-back">
+            TRULY TESTED
+          </h5>
+          <p>A cutting edge Benchmarking</p>
+          <p>approach for uparalleled</p>
+          <p>Database Speed</p>
+          <p className="font-semibold text-back">for vitess</p>
+        </div>
```

Comment:
I'd like to remove this entire div if you do not mind, I think this is a repetition to what we already have on the home page.

Reply:
I have removed it, the footer looked quite empty without it so I have only commented it out in the code as for now. 
We will along the way decide what can be put there :D